runtests.py

```diff
@@ -330,9 +330,9 @@ def build_project(args):
     cvars = distutils.sysconfig.get_config_vars()
     if 'gcc' in cvars['CC']:
         # add flags used as werrors tools/travis-test.sh
-        warnings_as_errors = (' -Werror=declaration-after-statement -Werror=vla'
+        warnings_as_errors = ('-Werror=declaration-after-statement -Werror=vla'
                               ' -Werror=nonnull -Werror=pointer-arith'
-                              ' -Wlogical-op')
+                              ' -Wlogical-op -Werror=unused-function ')
```

Comment:
I think this would be clearer written as
```
warnings_as_errors = ' '.join([
   '-Werror=declaration-after-statement',
   '-Werror=vla',
   '-Werror=nonnull',
   '-Werror=pointer-arith',
   '-Wlogical-op',
   '-Werror=unused-function',
])
env['CFLAGS'] = warnings_as_errors + ' ' + env.get('CFLAGS', '')
```